### ISSUE TITLE
docker support working again

### ISSUE THREAD

**Issue by despiegk** [threefoldtecharchive/jumpscaleX_archived#305]
/sandbox/code/github/threefoldtech/jumpscaleX/Jumpscale/tools/docker/Docker.py

need to be tested & fixed where required

**Comment by serboctor** [comment]
Depends on #501 and #468

**Comment by serboctor** [comment]
Duplicate of #468
